### DISCUSSION TITLE
[Projects Beta] adding tickets to a board fails silently

### DISCUSSION THREAD

**Discussion by markbirbeck** [community/community#16003] [Projects and Issues]
I'm not sure how I can describe this, since the board is private and the issue only arises with _one_ of our boards!

Anyway, I'll do the best I can. 😄 

First, we've been using the Projects (Beta) for a few months now, with no problems. We have 3 boards that have been working fine.

A few days ago we ceased being able to add tickets to one of our boards. However, we can still add tickets to our other boards.

The main, obvious difference between the problem board and the others is that it is our 'main' board, and consequently has many more tickets on it. One board has 94, but the problem board has 1200.

I've tried to archive some, but the action simply fails (i.e., the tickets just reappear in the column).

Related to that, we have 20 views on the problem board, but only 2 on the others.

Any thoughts?

**Comment by markbirbeck** [comment]
Ok...I just archived a single ticket and I am now allowed to add a new one to the board.

(So I have changed the issue title.)

**Comment by pawelru** [comment]
This is a duplicate of https://github.com/github/feedback/discussions/9678
Please upvote original post

**Reply by markbirbeck** [reply]
Mmm... kind of. I think I might change my title though, since for me there was no error message--I just happened to work out that there was a limit by experimenting.
